.github/workflows/release.yml

```diff
@@ -1,15 +1,7 @@
 name: kwctl release
-
-on:
-  push:
-    tags:
-    - 'v*'
```

Comment:
I guess you removed these constrains to debug building inside of your fork. We should bring them back now

Reply:
I was testing the GitHub actions in my own fork. I should probably have used another branch.